src/stores/tasks/task.ts

```diff
@@ -572,10 +588,10 @@ export class TaskRepo extends GenericRepo<CreateTaskOptions, UpdateTaskOptions,
     console.debug({ currentTask, pres, posts })
     const debugError = (x: any) => console.debug(x)
     for (let i = 0; i < pres.length; i++) {
-      await this.removePost(pres[i], task.id).catch(debugError)
+      this.removePost(pres[i], task.id)
     }
     for (let i = 0; i < posts.length; i++) {
-      await this.removePre(posts[i], task.id).catch(debugError)
+      this.removePre(posts[i], task.id)
```

Comment:
last I checked, unless we chain these together one after another, bad things happen.

Reply:
Probably, linter was saying it doesn't work as expected because it's not attached to a promise and thus does nothing. Actual fix is probably fixing it not being an async promise but I was lazy

Reply:
Let's pull a crowdstrike and see if production breaks